filament/src/Intersections.h

```diff
@@ -67,7 +67,7 @@ inline bool sphereConeIntersectionFast(
         math::float3 const& coneAxis,
         float coneSinInverse,
         float coneCosSquared) noexcept {
-    const math::float3 u = conePosition - math::float3(sphere.w * coneSinInverse) * coneAxis;
+    const math::float3 u = conePosition - coneAxis * sphere.w * coneSinInverse;
```

Comment:
we need parens: `coneAxis * (sphere.w * coneSinInverse)` -- to avoid extra unneeded muls.

Reply:
Yes, that makes sense, thanks, done.